gss-cb-signalling.xml

```diff
@@ -52,7 +52,7 @@ This Internet-Draft proposes the addition of a “channel bound” return flag f
       <t>
 The GSS-API <xref target="RFC2743"/> supports “channel binding” <xref target="RFC5056"/>, a technique for detection of man-in-the-middle (MITM) attacks in secure channels at lower network layers. This facility is meant to be all-or-nothing: either both the initiator and acceptor use it and it succeeds, or both must not use it. This has created a negotiation problem when retrofitting the use of channel binding into existing application protocols.</t>
         <t>
-However, GSS-APIv2u1 <xref target="RFC2743"/> does not specify channel binding behavior when only one party provides provides none. In practice some mechanisms ignore channel bindings when the acceptor provides none, but not when the initiator provides none. [XXX add references to the relevant SSPI docs? -Nico] Note that it would be useless to allow security context establishment to succeed when the initiator does not provide channel bindings but the acceptor does, at least as long as there's no outward indication of whether channel binding was used! Since the GSS-APIv2u1 does not provide any such indication, this document corrects that flaw.</t>
+However, GSS-APIv2u1 <xref target="RFC2743"/> does not specify channel binding behavior when only one party provides provides none.  Note that it would be useless to allow security context establishment to succeed when the initiator does not provide channel bindings but the acceptor does, at least as long as there's no outward indication of whether channel binding was used! Since the GSS-APIv2u1 does not provide any such indication, this document corrects that flaw.</t>
```

Comment:
I don't think the sentence before the parenthetical needs to be deleted, and the paragraph doesn't read right without it.  I believe the sentence (``In practice some mechanisms ignore channel bindings when the acceptor provides none, but not when the initiator provides none.``) describes the behavior of the krb5 mech as implemented everywhere.  I think Nico just thought there might be a reference to support that claim.